_inc/lib/debugger/class-jetpack-cxn-test-base.php

```diff
@@ -208,6 +285,8 @@ public static function failing_test( $name, $message, $resolution = false ) {
 			'pass'       => false,
 			'message'    => $message,
 			'resolution' => $resolution,
```

Comment:
Should we set a `'default'` in the `$resolution` switch statement above, now that each test will expect a small sentence when you click on the dropdown:

![image](https://user-images.githubusercontent.com/426388/55891278-92676000-5bb4-11e9-801d-de7b9efa8003.png)

Reply:
Since I'm looking at the message of the test above, I think we may want to add a space between the response and the response code?

Reply:
Default resolution step added in ce7c59bc6 .
Updated test-connection test to provide `: ` between the message and the response code in 4972f7052 .